webpack/assets/javascripts/react_app/components/charts/Chart.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Component from '../common/BindingComponent';
+
+export default class extends Component {
+    constructor(props) {
```

Comment:
The constructor isn't empty, it is continued on lines 6 and 7:

    constructor(props) {
        super(props);
    }